[DOCS] Clarifying remote clusters based on feedback from Support (#62335)
elastic/elasticsearch: docs/reference/modules/remote-clusters.asciidoc

[[modules-remote-clusters]]
== Remote clusters

> Set up _remote clusters_ to establish unidirectional
> connections required for
> <<xpack-ccr,{ccr}>> and <<modules-cross-cluster-search,{ccs}>>.

Reviewer: I'd unpack this a little bit. Maybe something like:
```suggestion
You can connect a local cluster to other {es} clusters, known as _remote
clusters_. Once connected, you can search remote clusters using
<<modules-cross-cluster-search,{ccs}>>. You can also sync data between clusters
using <<xpack-ccr,{ccr}>>.
```

> You configure a remote cluster and connect
> to a limited number of nodes in that remote cluster using either
> <<sniff-mode,sniff mode>> or <<proxy-mode,proxy mode>>.

Reviewer: I'd say "register" the remote cluster rather than "create" or "configure":

```suggestion
To register a remote cluster, the local cluster must connect to  nodes in the
remote cluster. You can do this using one of two connection modes:

* <<sniff-mode,sniff mode>>
* <<proxy-mode,proxy mode>>
```

Communication with a remote cluster uses the <<modules-transport,transport
layer>> to establish a number of <<long-lived-connections,long-lived>> TCP
connections from the coordinating nodes of the local cluster to the chosen

> nodes in the remote cluster.

Reviewer: This is one long sentence. I'd see if you could maybe break it up.

> Use the <<cluster-remote-info, remote cluster info API>> to retrieve
> information about the configured remote clusters, as well as the remote nodes
> that the node is connected to.

Reviewer: The imperative reads oddly to me here. I also don't think the clause about remote nodes is needed.
```suggestion
You can use the <<cluster-remote-info, remote cluster info API>> to get
information about registered remote clusters.
```

Sniff mode is the default connection mode.

[[gateway-nodes-selection]]
The _gateway nodes_ selection depends on the following criteria:

> * *version*: Remote nodes must be compatible with the cluster they are
> registered to, similar to the rules for
> <<rolling-upgrades>>:

Reviewer: ```suggestion
* *version*: Gateway nodes on the remote cluster must be compatible with the local cluster,
similar to the rules for <<rolling-upgrades>>:
```

| 7.1->7.x      | {no-icon} | {no-icon} | {no-icon} | {no-icon} | {yes-icon} | {yes-icon} | {yes-icon}
|====
// end::remote-cluster-compatibility-matrix[]
====

> * *role*: Dedicated master nodes never get selected.

Reviewer: ```suggestion
* *role*: Dedicated master nodes are never selected as gateway nodes.
```
